schemas/SuperProperties.json

```diff
@@ -15,21 +15,16 @@
       "type": "string"
     },
     "appPlatform": {
-      "description": "Used by web to identify the platform (Web Platform/Electron Platform).",
-      "type": "string"
-    },
-
-    "platformCodeName": {
       "description": "Used as a discriminant to breakdown usage per client.",
       "type": "string",
       "oneOf": [
-        {"const": "Web", "description": "Element Web platform code."},
-        {"const": "Desktop", "description": "Element Desktop platform code."},
-        {"const": "EI", "description": "Element iOS platform code."},
-        {"const": "EXI", "description": "Element-X iOS platform code."},
-        {"const": "EA", "description": "Element Android platform code."},
-        {"const": "EXA", "description": "Element-X Android platform code."},
-        {"const": "Other", "description": "Other Platform code."}
+        {"const": "Web Platform", "description": "Element Web platform."},
+        {"const": "Electron Platform", "description": "Element Desktop platform."},
+        {"const": "EI", "description": "Element iOS platform."},
+        {"const": "EXI", "description": "Element-X iOS platform."},
+        {"const": "EA", "description": "Element Android platform."},
+        {"const": "EXA", "description": "Element-X Android platform."},
+        {"const": "Other", "description": "Other Platform."}
       ]
     }
   },
```

Comment:
I suggest making changes to the properties definitions in a separate PR to the changes to the script, to make it easier for reviewers and future readers to see what is changing.

Reply:
Yes, you're right, but we do not have the bandwidth to ensure 150% quality on all our projects... Also adding a `const` with a space in this PR ensure that the change on the script generation works as expected.

I was tempted to tell @BillCarsonFr to better split their commits, for instance I do not like that commit on the script change also contains the generated files. It should be separate commit, to easily filter out generated commits.
But again let's not spend to much time on this.
A better commit history would have been:
- Update the script to support const with spaces
- Run the script
- Update appPlatform names
- Run the script

I guess this can be a better recipe for next time.

Reply:
I force pushed this branch to only contain the script changes with proper history
Script Change https://github.com/matrix-org/matrix-analytics-events/pull/105/commits/39eeb1c6c1d7c9d6ae6ac1996552e24cb83c4132

Run script to modify generated files https://github.com/matrix-org/matrix-analytics-events/pull/105/commits/641e66f7e5b8c97b332503046c1a442d63597a19


And there is a new PR to update `appPlatform` and remove `platfromCodeName` https://github.com/matrix-org/matrix-analytics-events/pull/106

Reply:
> Yes, you're right, but we do not have the bandwidth to ensure 150% quality on all our projects.

I think this is a bit like saying "we don't have time to write tests". You can save time in the short term, but in the longer term, breaking up changes into smaller steps that are easier to verify is an important way of avoiding bugs.